app/models/group.rb

```diff
@@ -97,7 +97,7 @@ def inactive?
   end
 
   def point_eligible_memberships
-    memberships.includes(:user).select { |m| m.end_date.nil? && m.archived.nil? }
+    memberships.includes(:user).select(&:active?)
```

Comment:
After the validations were introduced, point requests for newbies failed silently. Although listing newbies was incorrect here, so I fixed it.